app/scripts/modules/core/src/domain/IProject.ts

```diff
@@ -15,10 +17,17 @@ export interface IProjectConfig {
   clusters: IProjectCluster[];
 }
 
-export interface IProject {
+export interface IProject extends IIndexed {
+  config: IProjectConfig;
+  createTs?: number;
+  displayName?: string;
+  email: string;
+  href?: string;
```

Comment:
`href` doesn't actually exist on the types until they return in a search result; it feels a little dirty to mutate objects as a result of searching.